lib/Notification/bar/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { string } from 'prop-types';
+import { defaults, types } from './barTypes';
+import NotificationBarDanger from './NotificationBarDanger';
+import NotificationBarInformation from './NotificationBarInformation';
+import NotificationBarPromo from './NotificationBarPromo';
+import NotificationBarWarning from './NotificationBarWarning';
+
+const NotificationBar = ({ level, children, ...rest }) => {
+  const getNotificationBar = () => {
+    switch (level) {
+      case 'warning':
```

Comment:
💚 👍 Nice!

You could export these level strings as part of the component, and then use them in the app
```
// bar/index.js
...
NotificationBar.levels = {
  warning: 'warning',
  ...etc
}

// App.js

<NotificationBar level={NotificationBar.levels.warning} />
```